src/south/OPCUA/OPCUA.class.js

```diff
@@ -164,10 +221,15 @@ class OPCUA extends ProtocolHandler {
           intervalOpcEndTime = opcEndTime
         }
 
-        this.logger.silly(`Read from ${opcStartTime.getTime()} to ${intervalOpcEndTime.getTime()} the nodes ${nodesToRead}`)
+        // eslint-disable-next-line max-len
+        this.logger.silly(`Read from ${opcStartTime.getTime()} to ${intervalOpcEndTime.getTime()} (${intervalOpcEndTime - opcStartTime}ms) ${nodesToRead.length} nodes [${nodesToRead[0]}...${nodesToRead[nodesToRead.length - 1]}]`)
         // The request for the current Interval
         // eslint-disable-next-line no-await-in-loop
-        const dataValues = await this.session.readHistoryValue(nodesToRead, opcStartTime, intervalOpcEndTime)
+        /**
+         * @todo timeout and maxNodes should be parameters
```

Comment:
TODO can be removed. Seems like readTimeout and maxReturnValues are already parameters